app/workers/email_worker.rb

```diff
@@ -0,0 +1,25 @@
+class EmailWorker
+  include Sidekiq::Worker
+
+  def locale(user)
+    I18n.locale = user.locale || I18n.default_locale
+  end
+
+  def info(rent)
+    {
+      today: rent.created_at.strftime('%B %d, %Y'),
+      start: rent.from.strftime('%B %d, %Y'),
+      end: rent.to.strftime('%B %d, %Y'),
+      book: rent.book.title
+    }
+  end
+
+  def perform(rent_id)
+    rent = Rent.find(rent_id)
+    user = User.find(rent.user_id)
+    locale(user)
+    @receiver = user.email
```

Comment:
Also being picky here but if it's a variable that you only need in this method, an instance variable is overkill. I'd just do
`receiver = user.email`. Same goes for mail_info.

or even just in the ModelMailer line:
`ModelMailer.new_rent_notification(user.email, info(rent)).deliver`